addon/components/time-series-chart.js

```diff
@@ -787,7 +784,9 @@ export default ChartComponent.extend(LegendMixin, TimeSeriesLabelerMixin, Floati
     'yScale',
     'xValueDisplayName',
     'yValueDisplayName',
-    'hasAxisTitles',
+    'hasAxisTitles', // backward compatability support.
```

Comment:
You probably want to add jsdoc style here and have `deprecated` tag so that we can remove it later when we hit to version 2.0 for example.